src/editors/containers/VideoEditor/components/VideoSettingsModal/components/ThumbnailWidget/hooks.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { actions } from '../../../../../../data/redux';
+import * as constants from './constants';
+import * as module from './hooks';
+
+/** resampledFile({ canvasUrl, filename, mimeType })
+ * resampledFile takes a canvasUrl, filename, and a valid mimeType. The
+ * canvasUrl is parsed and written to an 8-bit array of unsigned integers. The
+ * new array is saved to  a new file with the same filename as the original image.
+ * @param {string} canvasUrl - string of base64 URL for new image canvas
+ * @param {string} filename - string of the original image's filename
+ * @param {string} mimeType - string of mimeType for the canvas
+ * @return {File} new File object
+ */
+export const createResampledFile = ({ canvasUrl, filename, mimeType }) => {
+  const arr = canvasUrl.split(',');
+  const bstr = atob(arr[1]);
+  let n = bstr.length;
+  const u8arr = new Uint8Array(n);
+  while (n--) {
+    u8arr[n] = bstr.charCodeAt(n);
+  }
+  return new File([u8arr], filename, { type: mimeType });
+};
+
+/** resampledImage({ image, filename })
+ * resampledImage takes a canvasUrl, filename, and a valid mimeType. The
+ * canvasUrl is parsed and written to an 8-bit array of unsigned integers. The
+ * new array is saved to  a new file with the same filename as the original image.
+ * @param {File} canvasUrl - string of base64 URL for new image canvas
+ * @param {string} filename - string of the image's filename
+ * @return {array} array containing the base64 URL for the resampled image and the file containing the resampled image
+ */
+export const resampledImage = ({ image, filename }) => {
```

Comment:
love the docstrings here, but why is this `resampledImage` not `resampleImage` if it is a function?